Assets/_BForBoss/_Core/Scripts/Managers/BaseWorldManager.cs

```diff
@@ -52,20 +54,29 @@ protected WeaponSceneManager WeaponSceneManager
 
         protected virtual void CleanUp()
         {
-            Debug.Log("Cleaning Up");
+            if (_environmentManager != null)
+            {
+                _environmentManager.CleanUp();
```

Comment:
If this is here, should we remove the same cleanup in the `WaveSandboxWorldManager` clean up?
https://github.com/PerigonGames/BForBoss/blob/c8de3e278ea85bed897070bf8fd74ace6d263eb9/Assets/_BForBoss/_Core/Scripts/Managers/WaveSandboxWorldManager.cs#L22

Reply:
i thought i removed it already. lemme ccheck

Reply:
Fixed